src/Jackett.Common/Definitions/polishtracker-api.yml

```
@@ -87,8 +91,17 @@ search:
       selector: id
     category:
       selector: category
-    title:
+    _language:
+      selector: language
+    _title_org:
+      selector: name
+    _title_pl:
       selector: name
+      filters:
+        - name: append
+          args: "{{ if .Config.polish_language }} POLISH{{ else }}{{ end }}"
+    title:
+      text: "{{ if eq .Result._language \"pl\" }}{{ .Result._title_pl }}{{ else }}{{ .Result._title_org }}{{ end }}"
```

```suggestion
      filters:
        - name: re_replace
          args: ["(?i)(\\.multi\\.)", ".{{ .Config.multilanguage }}."]
        - name: re_replace
          args: ["(?i)(\\.pl\\.)", ".{{ .Config.multilanguage }}."]
    title:
      text: "{{ if eq .Result._language \"pl\" }}{{ .Result.title_multilang }}{{ else }}{{ .Result.title_phase1 }}{{ end }}"
```
in line with other `multi` replacements

---

Thanks for the suggestion, but there is one problem with these replacements. In this approach, we will lost information on whether the release is actually multilanguage or has just polish language. This info is quite important for some people (like me), because someone can prefer really multilanguage rather than just polish. If it's possible, I would like to stay with my approach. What do you think about it?

---

https://github.com/Jackett/Jackett/pull/13566#discussion_r977712456 allows users to choose whether to replace `MULTI` (and `PL`) with `POLISH` or `MULTI.POLISH`. If you wish to retain `MULTI` in the title, then you would select the latter.

The only other results I saw with `pl` as the language had `POLISH` already in their titles, so there was no need to append it, and you would be able to distinguish these from `MULTI.POLISH` results.

---

Probably you have the wrong assumption. `PL` (and `POLISH`) releases are single language, so when I will select the second option, I will have `MULTI.POLISH` also for just Polish torrents (so `MULTI` there is not truth here). So I still won't distinguish between `PL` (single language) and `MULTI` (multi language). I hope that I better explained my point of view now :)

---

I've changed that now, see above.

Will need to do the same for polishsource after this.